reference/dic_tags.rst

```diff
@@ -540,14 +540,16 @@ kernel.event_listener
 
 **Purpose**: To listen to different events/hooks in Symfony
 
-This tag allows you to hook your own classes into Symfony's process at different
-points.
+During the execution of a Symfony application, different events are triggered.
+This tag allows you to *hook* your own classes into any of those events.
```

Comment:
We should add something that explains that you can also trigger your own events that will be dispatched through the same service as the core ones.